app/views/static_pages/contributing.html.erb

```diff
@@ -1,16 +1,34 @@
 <%= title('Contributing') %>
 
-<div class="container tmp-container-spacer">
-  <div class="page-intro">
-    <h1 class="page-heading-title">How to Contribute</h1>
-    <p class="page-intro__text">
-      The Odin Project is an Open Source project, built and maintained by volunteers who dedicate their time and skills to making The Odin Project one of the best free education platforms on the web. We are always working on projects to improve Odin and are always looking for people who want to join our growing team of maintainers.</a>
-    </p>
-  </div>
+<div class='page-container'>
+  <div class="grid grid-cols-12">
+    <div class="col-span-12 md:col-start-3 md:col-span-8">
+      <section class='text-center pb-12'>
+        <h1 class='page-heading-title'>How to Contribute</h1>
+        <p>
+          The Odin Project is an Open Source project, built and maintained by volunteers who dedicate their time
+          and skills to making The Odin Project one of the best free education platforms on the web. We are always
+          working on projects to improve Odin and are always looking for people who want to join our growing team
+          of maintainers.
+        </p>
+      </section>
 
-  <%= render 'static_pages/contributing/contributing_options' %>
+      <%= render 'static_pages/contributing/contributing_options' %>
+    </div>
+  </div>
 </div>
 
-<%= render 'static_pages/contributing/benefits' %>
+<section class="bg-gray-100 odin-dark-bg-accent">
+  <div class="container">
+    <div class="grid grid-cols-12">
+      <div class="col-span-12 md:col-start-3 md:col-span-8">
+        <div class="page-container">
```

Comment:
We should be able to remove this div with the page container above being in place.